src/containers/Charts/Chart.js

```diff
@@ -162,14 +163,18 @@ const Chart = ({ solarData }) => {
         </div>
       </div>
       <div className="savings-display-bar">
-        <p>Based on your annual energy usage, this solar system would offset {savingsValue}% per year! </p>
+        {history ?
```

Comment:
I like this setup here, that way the user knows what they're missing out on by skipping that previous form